lib/administrate/generator_helpers.rb

```diff
@@ -0,0 +1,7 @@
+module Administrate
+  module GeneratorHelpers
+    def call_generator(generator, *args)
+      Rails::Generators.invoke(generator, args, behavior: behavior)
```

Comment:
What do you think about passing `behavior` to `call_generator`? Otherwise, it seems like a mystery guest.